sympy/functions/elementary/exponential.py

```diff
@@ -230,6 +230,7 @@ def eval(cls, arg):
         from sympy.assumptions import ask, Q
         from sympy.calculus import AccumBounds
         from sympy.sets.setexpr import SetExpr
+        from sympy.functions.elementary.trigonometric import sin, cos
```

Comment:
[This line](https://github.com/sympy/sympy/blob/master/sympy/functions/elementary/exponential.py#L233) was added to master at exactly the same place, and GitHub cannot decide which one of the two lines should come first. Actually it does not matter, and you should take both in some order.